### PR TITLE
Updating holomap, adding it to the Horizon, adding it to cyborgs

### PR DESCRIPTION
**Pull request by Sindorman**
It saddens me that holomap was never updated and forgotten when we switched to Horizon. I think it is very useful for when people are new to the server, or even people who roleplay characters who are new to their position and the ship. So I updated the holomap and added it to the horizon hallways on all 3 decks.

changes:

- Updated holomap to work with horizon, updated its legend sprite, areas
- Mapped in holomaps into general hallways across all 3 decks
- Added "View Holomap" verb to the cyborgs to be able to view built-in holomap

Images of what it looks like:

dreamseeker_IyrByLqyWn
(Deck 1)

dreamseeker_TYeufrLMsl
(Deck 2)

dreamseeker_1CqhebF6SB
(Deck 3)

Cyborg using verb:
https://user-images.githubusercontent.com/25555314/206879755-661a82ca-c9b9-4b46-91c4-c176f1b82413.mp4

### CODE REVIEW

**Comment by Sindorman** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

File: code/game/machinery/station_holomap.dm [hunk 1]
```diff
@@ -185,6 +192,22 @@
 
 // TODO: Make these constructable.
 
+/obj/machinery/station_map/mobile
+	idle_power_usage = 0
+	active_power_usage = 0
```
Comment (SleepyGemmy) [inline code/game/machinery/station_holomap.dm:197]: `POWER_USE_OFF` macro?

File: code/game/machinery/station_holomap.dm [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /obj/machinery/station_map
-	name = "station holomap"
-	desc = "A virtual map of the surrounding station."
+	name = "holomap"
+	desc = "A virtual map of the surrounding deck."
```
Comment (SleepyGemmy) [inline code/game/machinery/station_holomap.dm:3]:
```suggestion
	desc = "A virtual map of the surrounding area."
```

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 3]
```diff
@@ -506,6 +507,22 @@
 	to_chat(src, SPAN_NOTICE("You [C.toggled ? "disable" : "enable"] [C.name]."))
 	C.toggled = !C.toggled
 
+/mob/living/silicon/robot/verb/view_holomap()
+	set category = "Robot Commands"
+	set name = "View Holomap"
+	set desc = "View Holomap of the current level."
```
Comment (SleepyGemmy) [inline code/modules/mob/living/silicon/robot/robot.dm:513]:
```suggestion
	set desc = "View a holomap of the current level."
```

**Review by SleepyGemmy — APPROVED** [review]

File: code/game/machinery/station_holomap.dm (hunk 1 above)
Comment (Sindorman) [reply code/game/machinery/station_holomap.dm:197]: That's for `use_power`, does not seem like just setting `user_power` to `POWER_USE_OFF` will stop it from using power.

**Pull request merged**